skimage/filters/tests/test_gaussian.py

```diff
@@ -19,6 +19,11 @@ def test_negative_sigma():
                  sigma=np.asarray([-1.0, 1.0]))
 
 
+def test_1D():
```

Comment:
What is this test supposed to test? Isn't it equivalent to sample from #5195? It doesn't fail with the previous version...